api_io_sign.yaml

```diff
@@ -0,0 +1,73 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: IO Sign - Backend API
+host: api-app.io.pagopa.it
+basePath: /api/v1/sign
+schemes:
+  - https
+security:
+  - Bearer: []
+paths:
+  "/qtsp/clauses/filled_document":
+    post:
+      operationId: createFilledDocument
+      summary: Create a filled document for QTSP.
+      description: |
+        The caller of this API is expected to poll on it since the endpoint
```

Comment:
How is the caller supposed to poll as this is a POST (which is suposed to not be idempotent)? Is there another endpoint or this statement is obsolete?

Reply:
ping :-)